### PR TITLE
Sssrose Create Beta

### PR DESCRIPTION
**Pull request by sssrose**
hot air

### CODE REVIEW

File: balloons [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Dont forget to check out the hot air balloons!!
+
+#and check out the micro-brews
```
Comment (swanchr) [inline balloons:3]: Can you recommend some?

File: balloons [hunk 2]
```diff
@@ -0,0 +1,3 @@
+### Dont forget to check out the hot air balloons!!
```
Comment (swanchr) [inline balloons]: Is that safe?

File: blank [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#Alb, NM
+
+##Places to eat
+
+##papa felipe's
+##Sadie's
+
+##Red or Green?
```
Comment (swanchr) [inline blank:8]: Why limit yourself to two colors?

Comment (sssrose) [reply blank:8]: Those are the two types of Chili in New Mexico @swanchr

File: balloons (hunk 1 above)
Comment (sssrose) [reply balloons:3]: 🍺 lol ...there is a beer emoji @swanchr

**Pull request merged**